client/src/components/PublicPages/AuditPlanner/AuditPlanCard.tsx

```diff
@@ -105,23 +119,24 @@ const AuditPlanCard: React.FC<IProps> = ({ disabled, electionResults }) => {
               { label: 'Batch Comparison', value: 'BATCH_COMPARISON' },
             ]}
             value={selectedAuditType}
+            vertical={isMobileWidth}
           />
         </Section>
 
         <Section>
           <SubHeading>Risk Limit</SubHeading>
-          <Slider
+          <RiskLimitSlider
             disabled={disabled}
             labelRenderer={value => (
               <span
                 // A hack to display the listed values on both the slider axis and the slider label,
                 // and all other values on only the slider label. More recent versions of the
                 // Blueprint slider actually support this differentiation
-                style={{
-                  color: [0, 5, 10, 15, 20].includes(value)
-                    ? undefined
-                    : 'white',
-                }}
+                className={
+                  ![0, 5, 10, 15, 20].includes(value)
+                    ? HIDDEN_LABEL_CLASS_NAME
+                    : undefined
+                }
```

Comment:
Here's the issue that I'm fixing here (note the streaks of white across the axis labels):

<img width="645" alt="issue" src="https://user-images.githubusercontent.com/12616928/193364016-7c2c92ec-24a1-4428-b9e9-ae7469edfe1c.png">